### PR TITLE
Monthly flow client side tidy up

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?
The current monthly contribution flow does not have the best design. Therefore, I am tidying it up. In addition, a new query parameter called: `contributionValue` is added.

## Changes
* Added monthly contribution parameter
* Deleted accordion
* Deleted some extra HTML elements
* Changed the monthly contributor's thank you page route from `/join/contributor/thankyou` to `/monthly-contribution/thankyou`
* Server side validation (£5 mini)
* Client-side validation for stripe button.

## Tested in

| Browser | Supporter checkout [Stripe] | Supporter checkout [Paypal] | Monthly Contribution [Stripe] |  Friend to Paid [Stripe] | Friend to Paid [Paypal]
|---------|-----------------|-----------------|----------------------|----------------------|----|
| IE         |            ✅              |        ✅                   |      ✅                   |       ✅                           |  
| Chrome  |         ✅              |        ✅                    |   ✅               |           ✅             |        ✅                

#### Step 1
mont1

#### Step 2
month2

#### Step 3
month3

#### Step 4
month4

#### Step 1
monthcheck

#### Step 2
month-permonth

#### Step 3
monththankyounew

#### No step 4 😀

### CODE REVIEW

File: frontend/assets/javascripts/src/modules/form/validation/validationProfiles.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ define([
     };
 
     /**
+     * @param contributionElem
+     * @returns {*}
+     */
+    var validContributiondValue = function (contributionElem) {
```
Comment (rupertbates) [inline frontend/assets/javascripts/src/modules/form/validation/validationProfiles.js]: Is this a typo 'validContributiondValue'? Should be validContributionValue?

Comment (svillafe) [inline frontend/assets/javascripts/src/modules/form/validation/validationProfiles.js]: Good spot :)

**Comment by svillafe** [comment]
It is working on PROD. Tested supporter's and contribution's  flow in Chrome.

**Comment by svillafe** [comment]
Supporter Stripe/Paypal and monthly contribution has been tested in PROD using IE. 🐟

**Pull request merged**